stitch/build.gradle

```diff
@@ -69,6 +69,10 @@ android {
         timeOutInMs 10 * 60 * 1000  // 10 minutes
         installOptions "-d","-t"
     }
+
+    packagingOptions {
+        pickFirst 'protobuf.meta'
```

Comment:
what did this do?

Reply:
Seems to have to do with the latest version of espresso: https://issuetracker.google.com/issues/69333598